metaflow/client/core.py

```diff
@@ -203,6 +213,9 @@ def default_namespace() -> str:
     return get_namespace()
 
 
+MetaflowArtifacts = NamedTuple
```

Comment:
This is just to give it a proper type so that there are no issues in the generated pyi file. It's not very informative but I don't think we can do much better.